pkg/webhooks/webhookpatch.go

```diff
@@ -187,3 +191,14 @@ func (w *WebhookCertPatcher) startCaBundleWatcher(stop <-chan struct{}) {
 		}
 	}
 }
+
+func getInjectionWebhookConfigName(revision string, namespace string) string {
```

Comment:
suppose we can combine the `getInjectionWebhookConfigName` and  `getValidationWebhookConfigName` into one method?

Reply:
Done